Equilobe.DailyReport.Models/Storage/Policy.cs

```diff
@@ -16,9 +17,18 @@ public class Policy
 
         public SourceControlOptions SourceControlOptions { get; set; }
 
-        public GeneratedProperties GeneratedProperties { get; set; }
-
         [XmlIgnore]
         public List<string> EmailCollection { get; set; }
+
+        public string RootPath { get; set; }
+
+        [XmlIgnore]
+        public string LogPath { get { return Path.Combine(RootPath, "Logs"); } }
```

Comment:
atentie cu tot ce inseamna "Policy", deoarece in Models/Web a nu se umbla decat daca lucrezi pe web - alea implementeaza interfete ce sunt strict legate de DB si cu ajutorul carora se face bind-ul de date si sync-ul dintre DB si UI [in ambele sensuri]